fast_pytorch_kmeans/kmeans.py

```diff
@@ -22,6 +23,9 @@ class KMeans:
 
     mode: {'euclidean', 'cosine'}, default: 'euclidean'
       Type of distance measure
+      
+    init_method: {'random', 'point', '++'}

```

Comment:
fast-pytorch-kmeans is intended to have a similar interface as [sklearn.cluster.KMeans](https://scikit-learn.org/stable/modules/generated/sklearn.cluster.KMeans.html#sklearn-cluster-kmeans), so it would be better to name this argument as well as it's possible options to be close to sklearn's. for example:
`init: {'gaussian', 'random', 'k-means++'}`
where 'random' here refers to randomly choosing K points from data